c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl

```diff
@@ -131,6 +131,13 @@ Version 1.6.3
 
      configfile = "${directory}/${'development' if development == 'TRUE' else 'production'}.ini"
 
+12. On deploy the post-restore-code will be moved to pre-restore-code and deeply modified.
+    It you don't modify it just get the new version:
```

Comment:
"If you did not modify it you can just copy the new version with the following:"